addons/mail/static/src/core/web/messaging_menu.js

```diff
@@ -224,7 +225,7 @@ export class MessagingMenu extends Component {
         if (this.canPromptToInstall) {
             value++;
         }
-        if (this.notification.permission === "prompt") {
+        if (this.notification.permission === "prompt" && !isIOS()) {
```

Comment:
not a big deal, but prone to errors to have duplicate code instead of a single getter for both `this.notification.permission === "prompt" && !isIOS()`